packages/contracts-bedrock/test/kontrol/proofs/utils/KontrolUtils.sol

```diff
@@ -0,0 +1,154 @@
+        kevm.symbolicStorage(address(ghostBytes));
+
+        /* Load the size encoding into the first slot of ghostBytes*/
+        vm.store(address(ghostBytes), bytes32(uint256(0)), bytes32(bytesSlotValue));
+
+        sBytes = ghostBytes.ghostBytes();
+    }
+
+    /// @dev Creates a bounded symbolic bytes[] memory representing a withdrawal proof
+    /// Each element is 17 * 32 = 544 bytes long, plus ~10% margin for RLP encoding: each element is 600 bytes
+    /// The length of the array to 10 or fewer elements
+    function freshWithdrawalProof() public returns (bytes[] memory withdrawalProof) {
+        /* Assume arrayLength = 2 for faster proof speeds */
+        uint256 arrayLength = 10;
+
+        withdrawalProof = new bytes[](arrayLength);
+
+        /* Deploy ghost contract */
+        GhostBytes10 ghostBytes10 = new GhostBytes10();
+
+        /* Make the storage of the ghost contract symbolic */
+        kevm.symbolicStorage(address(ghostBytes10));
+
+        /* Each bytes element will have a length of 600 */
+        uint256 bytesSlotValue = 600 * 2 + 1;
+
+        /* Load the size encoding into the first slot of ghostBytes*/
+        vm.store(address(ghostBytes10), bytes32(uint256(0)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(1)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(2)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(3)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(4)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(5)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(6)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(7)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(8)), bytes32(bytesSlotValue));
+        vm.store(address(ghostBytes10), bytes32(uint256(9)), bytes32(bytesSlotValue));
+
+        /* Current approach */
+
+        withdrawalProof = ghostBytes10.getGhostBytesArray();
+
+        /* Second approach */
+
+        /* withdrawalProof[0] = ghostBytes10.ghostBytes0(); */
+        /* withdrawalProof[1] = ghostBytes10.ghostBytes1(); */
+        /* withdrawalProof[2] = ghostBytes10.ghostBytes2(); */
+        /* withdrawalProof[3] = ghostBytes10.ghostBytes3(); */
+        /* withdrawalProof[4] = ghostBytes10.ghostBytes4(); */
+        /* withdrawalProof[5] = ghostBytes10.ghostBytes5(); */
+        /* withdrawalProof[6] = ghostBytes10.ghostBytes6(); */
+        /* withdrawalProof[7] = ghostBytes10.ghostBytes7(); */
+        /* withdrawalProof[8] = ghostBytes10.ghostBytes8(); */
+        /* withdrawalProof[9] = ghostBytes10.ghostBytes9(); */
+
+        /* First approach */
+
+        /* for (uint256 i = 0; i < withdrawalProof.length; ++i) { */
+        /*     withdrawalProof[i] = freshBigBytes(600);
+        /* } */
```

Comment:
Is there any tradeoff between these approaches or should we delete the second one?
```suggestion
        withdrawalProof = ghostBytes10.getGhostBytesArray();
```

Reply:
Right now the tradeoffs are not very clear, so I've kept all the explored options while this is figured out

Reply:
I'm going to resolve this. Eventually, these should be removed, but not yet, so I'm leaving them for now. Let me know if you have any concerns with this! I believe the first approach should be faster, but I would like to keep the others while we don't have a final solution if that's alright